avogadro/qtplugins/label/label.cpp

```diff
@@ -131,19 +143,74 @@ struct LayerLabel : Core::LayerData
                        SLOT(setRadiusScalar(double)));
       form->addRow(QObject::tr("Distance from center:"), spin);
 
-      v->addLayout(form);
-
-      // residue or atoms?
-      QCheckBox* check = new QCheckBox(QObject::tr("Atom Labels"));
-      check->setChecked(atomLabel);
-      QObject::connect(check, &QCheckBox::clicked, slot, &Label::atomLabel);
-      v->addWidget(check);
-
-      check = new QCheckBox(QObject::tr("Residue Labels"));
-      check->setChecked(residueLabel);
-      QObject::connect(check, &QCheckBox::clicked, slot, &Label::residueLabel);
-      v->addWidget(check);
+      QComboBox* atom = new QComboBox;
+      atom->setObjectName("atom");
+      for (char i = 0x00; i < std::pow(2, 4); ++i) {
+        if (i == 0) {
+          atom->addItem(QObject::tr("None"), QVariant(LabelOptions::None));
+        } else {
+          char val = 0x00;
```

Comment:
Just use a string list. I know this works in English, but it won't work for translation in many languages.

QStringList options;
options <<  tr("None") << tr("Index") << tr("Element Symbol") << tr("Symbol + Index"); // etc